src/client/Microsoft.Identity.Client/ManagedIdentity/ImdsManagedIdentitySource.cs

```diff
@@ -82,6 +82,9 @@ protected override ManagedIdentityRequest CreateRequest(string resource)
                     break;
             }
 
+            // uncomment in follow-up IMDS retry policy PR
```

Comment:
nit: I recommend you remove this. It's dead code. The new PR will do the right thing.

Reply:
done